iree/rt/api.cc

```diff
@@ -244,9 +244,16 @@ iree_rt_module_lookup_function_by_ordinal(iree_rt_module_t* module,
     return IREE_STATUS_INVALID_ARGUMENT;
   }
 
-  IREE_API_ASSIGN_OR_RETURN(
-      auto function, handle->LookupFunctionByOrdinal(
-                         static_cast<Function::Linkage>(linkage), ordinal));
+  auto function_or = handle->LookupFunctionByOrdinal(
```

Comment:
ahh good catch - could just change the method to return the right thing (I can do that later)